include/cutlass/arch/memory_sm75.h

```diff
@@ -56,30 +56,57 @@ inline __device__ void ldsm(Array<unsigned, MatrixCount> & D, void const* ptr);
 //
 /////////////////////////////////////////////////////////////////////////////////////////////////
 
-#if (__CUDACC_VER_MAJOR__ == 10 && __CUDACC_VER_MINOR__ >= 2) || (__CUDACC_VER_MAJOR__ >= 11)
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifdef __clang__
+  //  __cvta_generic_to_shared was added in Clang 14: https://reviews.llvm.org/D111665
+  #define CUTLASS_CLANG_SUPPORTS_CVTA_GENERIC_TO_SHARED (__clang_major__ >= 14)
+
+  // __nvvm_get_smem_pointer added in Clang 14: https://reviews.llvm.org/D111665
+  // ... but broken on Windows until Clang 15: https://reviews.llvm.org/D122897
+  #ifndef _WIN32
+    #define CUTLASS_CLANG_SUPPORTS_NVVM_GET_SMEM_POINTER (__clang_major__ >= (defined(_WIN32) ? 15 : 14))
+  #endif
 
-#if defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 750)
-#define CUDA_LDMATRIX_ACTIVATED 1
+  // ldmatrix PTX instructions added in Clang 14: https://reviews.llvm.org/D107046
+  // ... but broken until Clang 15:
+  //   * https://reviews.llvm.org/D121666
+  //   * https://reviews.llvm.org/D126846
+  #define CUTLASS_CLANG_SUPPORTS_LDMATRIX (__clang_major__ >= 15)
 #endif
 
-#define CUDA_LDMATRIX_SUPPORTED 1
+#ifdef __NVCC__
```

Comment:
nvrtc is not included.

Reply:
Replaced with `__CUDACC__` as per [NVCC](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/#nvcc-identification-macro) and [NVRTC](https://docs.nvidia.com/cuda/nvrtc/index.html#predefined-macros) documentation.